src/Lexers/StatementLexer.php

```diff
@@ -14,6 +14,7 @@
 use Blueprint\Models\Statements\SendStatement;
 use Blueprint\Models\Statements\SessionStatement;
 use Blueprint\Models\Statements\ValidateStatement;
+use Illuminate\Support\Arr;
 use Illuminate\Support\Str;
 
 class StatementLexer implements Lexer
```

Comment:
In `analyze` simply allow matching of `send-*`, `fire-*`, etc.

Reply:
What's your preferred way of handling that? Simply `'fire', 'fire-*' => $this->analyzeEvent($statement)` obviously won't work since a `match` statement on a literal string is used.

Reply:
Went with `fnmatch` since that allows easy glob matching.